bears/java/CheckstyleBear.py

```diff
@@ -10,6 +10,13 @@
     "geosoft": "http://geosoft.no/development/geosoft_checks.xml"}
 
```

Comment:
regarding commit body: use "Adds" instead of "Add" (NOT in the shortlog, but in the body!)

also, end sentences with period please